gitdm/serializers.py

```
@@ -0,0 +1,15 @@
+from .models import Patient, User
+
+
+class CustomTokenObtainPairSerializer(TokenObtainPairSerializer):
+    username_field = 'email'
+
+
+class PatientSerializer(serializers.ModelSerializer):
+    primary_doctor_id = serializers.IntegerField(source='primary_doctor.id', read_only=True)
+    class Meta:
+        model = Patient
+        fields = ['id', 'national_id', 'full_name', 'dob', 'sex', 'primary_doctor', 'primary_doctor_id', 'created_at']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)